scripts/build_funcs.sh

```diff
@@ -6,10 +6,9 @@ PS4='\033[1m[$0:${LINENO}] $\033[0m '
 
 BINARY=sonobuoy
 TARGET=sonobuoy
-GOTARGET=github.com/vmware-tanzu/$TARGET
+GOTARGET=github.com/vmware-tanzu/"$TARGET"
```

Comment:
should this be quotes or `${TARGET}`?

Reply:
Even with the brackets shellcheck will want quotes. The brackets can, in some cases, make it more readable because it maks it stand out a bit more. The quotes are more important because of word breaking and such.